src/commands/build/watch/watcher.rs

```diff
@@ -18,15 +18,12 @@ pub fn wait_for_changes(
         match get_changed_path_from_event(event) {
             Ok(Some(path)) => {
                 message::working("Detected changes...");
-                //wait for cooldown
-                while let Ok(_) = rx.recv_timeout(cooldown) {
-                    message::working("Detected change during cooldown...");
```

Comment:
Maybe the logs were useful and we can turn them into debug logs?

Reply:
I chatted with @ashleymichal and she said that these logs initially made sense for smaller projects (they make users feel like wrangler watch is actively working), but just became too verbose when editing entire directories (e.g. large webpack workers). I think it's worth keeping these messages out for now, unless we come up with more detailed/meaningful logs to replace them.